python_modules/dagster-graphql/dagster_graphql/schema/schedules/__init__.py

```diff
@@ -102,6 +104,8 @@ class Arguments:
     class Meta:
         name = "StopRunningScheduleMutation"
 
+    @capture_error
+    @check_permission(Permissions.STOP_RUNNING_SCHEDULE)
```

Comment:
should we push our luck and merge these into EDIT_SCHEDULE :)

Reply:
happy to do that after the incident review tomorrow